layout/hp-markup.tex

```diff
@@ -111,18 +145,18 @@
 
 % Text break made of \Stars (only used to define other commands)
 \makeatletter
-\def\sbre@k{\mbox{}\nobreak\hfill\mbox{}\allowbreak\rule{.60\textwidth}{.0pt}\par%
+\def\sbreak{\mbox{}\nobreak\hfill\mbox{}\allowbreak\rule{.60\textwidth}{.0pt}\par%
```

Comment:
This macro name is deliberate, so that it can't accidentally be used in normal text. Did you need to use it in normal text?

Reply:
didn't know this, reverted